packages/webgpu/react-native-wgpu.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => min_ios_version_supported, :visionos => "1.0" }
+  s.platforms    = { :ios => min_ios_version_supported, :visionos => "1.0", :osx => "11.0" }
```

Comment:
Actually, I should revert this. The rest of the sources don't compile for macOS yet.